Core/Core/Dashboard/K5/View/Grades/K5GradesView.swift

```diff
@@ -68,13 +69,17 @@ struct K5GradesView: View {
                             .font(.bold24)
                             .foregroundColor(.licorice)
                     })
+                    .accessibility(label: Text("Select grading period", bundle: .core))
```

Comment:
I would move the state announcement to this label. Currently, if I double tap on the selector it just keeps announcing "Select grading period". It would be better to announce the state upon double tapping as well. "Select grading period, closed", "Select grading period, opened".